.circleci/config.yml

```diff
@@ -1,17 +1,17 @@
 version: 2
 
-xcode_version: &xcode_version 9.4
+xcode_version: &xcode_version 10.0
 iphone_name: &iphone_name iPhone 8
-ios_version: &ios_version 11.4
+ios_version: &ios_version 12.0
 opentok_version: &opentok_version 2.10.2
 fabric_sdk_version: &fabric_sdk_version 3.10.5
-preload_simulator: &preload_simulator xcrun instruments -w 'iPhone 8 (11.4) [2437FE39-2885-482D-A42F-A169C65AE271]' || true
+preload_simulator: &preload_simulator xcrun instruments -w 'iPhone 8 (12.0) [2437FE39-2885-482D-A42F-A169C65AE271]' || true
```

Comment:
We don't use this preloading anymore, you can try remove it and see if it works for you. The problem is that the identifier changes a lot so it breaks every time Circle deploys a new version of Xcode.